docs/api.md

```diff
@@ -496,6 +496,89 @@ await webcontainerInstance.fs.writeFile(
 );
 ```
 
+### ▸ `watch`
+
+Watch for changes to a given file or directory.
+
+<h4 id="watch-signature">
+  <a id="watch-signature">Signature</a>
+  <a href="#watch-signature" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+`watch(path: string, options: Options, listener: Listener): Watcher`
+
+`watch(path: string, listener: Listener): Watcher`
+
+<h4 id="watch-options">
+  <a id="watch-options"><code>Options</code></a>
+  <a href="#watch-options" class="header-anchor" aria-hidden="true">#</a>
+</h4>
+
+```ts
+interface Options {
+  encoding?: BufferEncoding | null;
+  persistent?: boolean;
```

Comment:
Can we add a description for this as well and explain what it does? I think we are missing a sub-section about this.

Reply:
we actually decided to remove it: not sure if you agree: https://github.com/stackblitz/webcontainer-docs/pull/47#discussion_r1373004645

Reply:
Oh I missed that comment. Yea I think it doesn't make much sense because, like you said, it's called from the main thread so there's nothing that keeps running as long as files are being watched cause it's not a separate process after all and it's just the main thread.

Reply:
So I agree that we should remove this option.